### PR TITLE
Add Spanish translation 🇪🇸

### PR DESCRIPTION
**Pull request by Reonu**
Arriba españa hostia ya 🇪🇸 🇪🇸 🇪🇸 🇪🇸 🇪🇸 🇪🇸

### CODE REVIEW

File: levels/menu/leveldata.c [hunk 1]
```diff
@@ -1933,6 +1933,19 @@ const Gfx dl_menu_texture_kurs_upper[] = {
     gsSPEndDisplayList(),
 };
 #endif
+
+#ifdef ENABLE_SPANISH
+// 0x0700DDA0 - 0x0700EDA0
```
Comment (thecozies) [inline levels/menu/leveldata.c:1938]: this isn't a real address, the other comments are decomp labeling. you can remove this and the Nivel one

File: levels/menu/leveldata.c [hunk 2]
```diff
@@ -1933,6 +1933,19 @@ const Gfx dl_menu_texture_kurs_upper[] = {
...
+ALIGNED8 static const Texture texture_menu_nivel_upper[] = {
+#include "levels/menu/main_menu_seg7.nivel_upper.rgba16.inc.c"
+};
+
+// 0x0700FF10 - 0x0700FF20
```
Comment (thecozies) [inline levels/menu/leveldata.c:1943]: dis

File: src/game/hud.c [hunk 3]
```diff
@@ -446,7 +446,8 @@ LangArray textTime = DEFINE_LANGUAGE_ARRAY(
     "TIME %0d'%02d\"%d",
     "TEMPS %0d'%02d\"%d",
     "ZEIT %0d'%02d\"%d",
-    "TIME %0d'%02d\"%d");
+    "TIME %0d'%02d\"%d",
+    "TIEM.%0d'%02d\"%d");
```
Comment (thecozies) [inline src/game/hud.c:451]: French has five letters and a space, should this be `TIEM. ` (space after period)

Comment (Reonu) [reply src/game/hud.c:451]: that makes it move into the HP meter slightly (but the same thing happens with french)

idk what do you prefer?

imagen

imagen

Comment (Reonu) [reply src/game/hud.c:451]: i guess it's fine if french does it, will change it

Comment (Reonu) [reply src/game/hud.c:451]: done

Comment (thecozies) [reply src/game/hud.c:451]: so you're saying anything is fine if french does it 👀

File: levels/menu/leveldata.c (hunk 1 above)
Comment (thecozies) [reply levels/menu/leveldata.c:1936]: @Reonu you removed it from German lmao

File: src/game/hud.c (hunk 3 above)
Comment (Reonu) [reply src/game/hud.c:451]: death

File: levels/menu/leveldata.c (hunk 1 above)
Comment (Reonu) [reply levels/menu/leveldata.c]: no idea what you're talking about

**Review by thecozies — APPROVED** [review]

File: src/game/ingame_menu.c [hunk 4]
```diff
@@ -56,8 +56,14 @@ extern u8 course_name_table_jp[];
...
 
+#ifdef ENABLE_SPANISH
+extern u8 dialog_table_es[];
+extern u8 course_name_table_es[];
+extern u8 act_name_table_es[];
+#endif
+
 // The language table for the game's dialogs, level names and act names.
-void *languageTable[][3] = {
+void *languageTable[][4] = {
```
Comment (arthurtilly) [inline src/game/ingame_menu.c:66]: this isnt right, the dimension is the width of each language entry (which is 3) not the length of the array

**Comment by Reonu** [comment]
Pointed to 2.1.0 directly. This is now a combined PR to add ascii, multilang, and Spanish.

**Comment by Reonu** [comment]
review @gheskett

**Comment by axollyon** [comment]
Repointed to develop/2.1.0-ascii again as we have a new secure PR for base ASCII

**Comment by Reonu** [comment]
ESPAÑA NÚMERO 1 🇪🇸 🇪🇸 🇪🇸 🇪🇸 ARRIBA ESPAÑA 🇪🇸 🇪🇸 🇪🇸 🇪🇸 🇪🇸 🇪🇸 🇪🇸

**Pull request merged**